src/Client.php

```diff
@@ -153,7 +156,7 @@ public function patchRaw(string|UriInterface $uri, array|\JsonSerializable $payl
         return $this->requestRaw('PATCH', $uri, $payload);
     }
 
-    public function postRaw(string|UriInterface $uri, array|\JsonSerializable $payload = [], array|HasParameters $parameters = []): ResponseInterface
+    public function postRaw(string|UriInterface $uri, array|\JsonSerializable|null $payload = [], array|HasParameters $parameters = []): ResponseInterface
```

Comment:
There might be a case in future where empty `[]` is preferred over empty body. If we convert `[]` to empty body now, it may require a breaking change later.

Reply:
As agreed, we will change the default `$payload` to be `null` in the next major release